NARFO_BE/ControllerUnitTests/UnitTest1.cs

```diff
@@ -45,7 +45,7 @@ public Tests()
         public async Task GetAllMembers_ShouldReturnAllMembers()
         {
             // Create Member object
-            var newMember = new Members { Id = 1, FirstName = "Mike", LastName = " Jack", Username = "MikeJack", Password = "Password1" };
+            var newMember = new _Member { Id = 1, Firstname = "Mike", SURNAME = " Jack", Username = "MikeJack", Password = "Password1" };
```

Comment:
I think this can be made multi line.

Reply:
thank you for your view ( :

Reply:
for testing purposes.. theres already a Member model to be imported using Entity Framework 👍